spec/classes/sensu_rabbitmq_spec.rb

```diff
@@ -93,6 +93,24 @@
       ) }
     end # when using local key
 
+    context 'when passing password as Sensitive' do
```

Comment:
Sure, this was the first quick one, will add the others once defined the best approach